docs/topics/proc_configuring-your-application-to-use-spring-boot.adoc

```diff
@@ -1,16 +1,15 @@
 [id='configuring-your-application-to-use-spring-boot_{context}']
 = Configuring your application to use {SpringBoot}
 
-To configure your application to use {SpringBoot}:
+Reference the `pom.xml` file in the {SpringBoot} BOM (Bill of Materials) artifact to configure your application.
```

Comment:
I'm sorry, I don't know how this slipped past me. You reference the BOM in the `pom.xml` file, not the other way around.

Reply:
I fixed this. Let me know what you think.